### ISSUE TITLE
ADDON_ACTION_BLOCKED within MultiBarBottomLeftButton7:SetAttribute()

### ISSUE THREAD

**Issue by frozn** [frozn/TipTac#140]
Source: https://www.curseforge.com/wow/addons/tiptac-reborn?comment=61

**pheeve:**
```
1x [ADDON_ACTION_BLOCKED] AddOn 'TipTac' tried to call the protected function 'MultiBarBottomLeftButton7:SetAttribute()'.
[string "@Interface/AddOns/!BugGrabber/BugGrabber.lua"]:480: in function <Interface/AddOns/!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface/FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@Interface/FrameXML/ActionButton.lua"]:410: in function `Update'
[string "@Interface/FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@Interface/FrameXML/ActionButton.lua"]:795: in function `OnEvent'
[string "@Interface/FrameXML/ActionButton.lua"]:216: in function <Interface/FrameXML/ActionButton.lua:213>

Locals:
Skipped (In Encounter)
```

**Comment by Nukme** [comment]
same type lua error

```
1x [ADDON_ACTION_BLOCKED] 插件 'TipTac' 尝试调用保护功能 'MultiBarLeftButton12:SetAttribute()'。
[string "@Interface/AddOns/!BugGrabber/BugGrabber.lua"]:480: in function <Interface/AddOns/!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@Interface/FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@Interface/FrameXML/ActionButton.lua"]:410: in function `Update'
[string "@Interface/FrameXML/ActionButton.lua"]:884: in function `OnEvent'
[string "@Interface/FrameXML/ActionButton.lua"]:264: in function <Interface/FrameXML/ActionButton.lua:256>

Locals:
Skipped (In Encounter)
```

**Comment by frozn** [comment]
Source: https://www.curseforge.com/wow/addons/tiptac-reborn?comment=71

**Kalec_Stromhir:**
same issue with using dominos.

```
1x [ADDON_ACTION_BLOCKED] AddOn 'TipTac' tried to call the protected function 'DominosActionButton29:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:410: in function <FrameXML/ActionButton.lua:373>
[string "=[C]"]: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:1177: in function <FrameXML/ActionButton.lua:1174>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:266: in function <FrameXML/SecureHandlers.lua:263>
[string "=[C]"]: ?
[string "@FrameXML/SecureHandlers.lua"]:322: in function <FrameXML/SecureHandlers.lua:306>
[string "=(tail call)"]: ?
```

**Comment by limpieza** [comment]
Same Issue : Using blizzard bar addon : 

1x [ADDON_ACTION_BLOCKED] AddOn 'TipTac' tried to call the protected function 'MultiBar5Button2:SetAttribute()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/ActionButton.lua"]:350: in function `UpdatePressAndHoldAction'
[string "@FrameXML/ActionButton.lua"]:410: in function `Update'
[string "@FrameXML/ActionButton.lua"]:362: in function `UpdateAction'
[string "@FrameXML/ActionButton.lua"]:795: in function `OnEvent'
[string "@FrameXML/ActionButton.lua"]:216: in function <FrameXML/ActionButton.lua:213>

**Comment by zaphon** [comment]
I'm getting this same error and no idea why it's even saying it's blaming this on TipTac.  This feels like something else is getting tainted which is causing it to taint all of this.

**Comment by laki28** [comment]
[taint.zip](https://github.com/frozn/TipTac/files/10090435/taint.zip)
I have attached the related log. An intermittent error occurred during the combat.

**Comment by frozn** [comment]
Should be fixed with latest release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release). Please try again with this version.
